scripts/tenderly_reset.ts

```diff
@@ -0,0 +1,92 @@
+// Run this only when you have NEW Tenderly FORK and what to do a new SETUP for it.
+
+import 'hardhat-deploy-ethers';
+import '@nomiclabs/hardhat-etherscan';
+import { ethers, getNamedAccounts, network } from 'hardhat';
+import { networkConfig } from '../helper-hardhat-config';
+
+import { unlink, rm } from 'fs';
+import { promisify } from 'util';
+import axios from 'axios';
+
+const unlinkAsync = promisify(unlink);
+const rmAsync = promisify(rm);
+
+const filesToDelete = [
+  'deployments/tenderly/PostageStamp.json',
+  'deployments/tenderly/PriceOracle.json',
+  'deployments/tenderly/Redistribution.json',
+];
+const directoryToDelete = 'deployments/tenderly/solcInputs';
+
+async function deleteFiles(filePaths: string[]) {
+  try {
+    for (const filePath of filePaths) {
+      await unlinkAsync(filePath);
+      console.log(`Deleted file: ${filePath}`);
+    }
+    console.log('All files have been deleted successfully.');
+  } catch (error) {
+    console.error('Error deleting files');
+  }
+}
+
+async function deleteDirectory(directoryPath: string) {
+  try {
+    await rmAsync(directoryPath, { recursive: true, force: true });
+    console.log(`Deleted directory and all its contents: ${directoryPath}`);
+  } catch (error) {
+    console.error('Error deleting directory:', error);
+  }
+}
+
+async function main() {
+  const { deployer } = await getNamedAccounts();
+
+  // Remove previous deployment so we start fresh for contracts that will be deployed, modify them per case
+  await deleteFiles(filesToDelete);
+  await deleteDirectory(directoryToDelete);
+
+  const forkId = network.config.url.split('/').slice(-2).join('/');
```

Comment:
> Property 'url' does not exist on type 'NetworkConfig'.

Reply:
yeah, it does exist but type is
export type NetworkConfig = HardhatNetworkConfig | HttpNetworkConfig;

I made type check now.